Docs/src/tutorials/basictutorials/basictutorial1.md

```diff
@@ -0,0 +1,302 @@
+# Basic Tutorial 1 {#basictutorial1}
+
```

Comment:
we should keep the "TUTORIAL INTRODUCTION" text from the wiki that gives an outline.

Reply:
Can you clarify please?

Reply:
![bildschirmfoto vom 2017-09-10 19-45-28](https://user-images.githubusercontent.com/4941802/30251506-b353591a-9660-11e7-9bb9-173436298b8a.png)

Reply:
I see. I got this! 
Will be done.